build.gradle

```diff
@@ -44,6 +44,8 @@ dependencies {
     String jUnitVersion = '5.4.0'
     String javaFxVersion = '11'
 
+    compile 'org.apache.commons:commons-text:1.9'
```

Comment:
I think we might need to get permission from prof using the forum for using additional external library like this

Reply:
yep ive submitted a request, waiting for approval

Reply:
ok it has been approved

Reply:
thank you man